framework/src/transfers/MultiAppMeshFunctionTransfer.C

```diff
@@ -140,8 +129,12 @@ MultiAppMeshFunctionTransfer::transferVariable(unsigned int i)
           {
             if (bboxes[i_from].contains_point(*node + _to_positions[i_to]))
             {
+              // <system id, node id>
               std::pair<unsigned int, dof_id_type> key(i_to, node->id());
+              // <pid, <key>> --> point id
```

Comment:
The angle brackets around `key` confuses me. If I see angle brackets, I expect to see template arguments

Reply:
I would not use `<>` to denote template arguments (I think?) and `-->` to denote mapping on the same line. There is too much `>`. I would write out what you are trying to communicate here